### PR TITLE
Merge back release 6.15.1 and 6.15.0

### PR DESCRIPTION
**Pull request by tamaskozmer**

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

File: apps/student/src/main/java/com/instructure/student/features/elementary/course/ElementaryCoursePagerAdapter.kt [hunk 1]
```diff
@@ -16,19 +16,80 @@
...
+
+        progressBar.setVisible()
+        setupViews(webView, progressBar)
+        webView.loadUrl(tabs[position].url)
+
+        return view
+    }
+
+    override fun destroyItem(container: ViewGroup, position: Int, item: Any) {
+        container.removeView(item as View)
```
Comment (kdeakinstructure) [inline apps/student/src/main/java/com/instructure/student/features/elementary/course/ElementaryCoursePagerAdapter.kt:57]: I think position is unnecessary parameter here.

File: apps/student/src/main/java/com/instructure/student/features/elementary/course/ElementaryCoursePagerAdapter.kt [hunk 2]
```diff
@@ -16,19 +16,80 @@
...
+        webView.setZoomSettings(false)
+        webView.canvasWebViewClientCallback = object : CanvasWebView.CanvasWebViewClientCallback {
+            override fun openMediaFromWebView(mime: String, url: String, filename: String) {
+                RouteMatcher.openMedia(activity, url)
+            }
+
+            override fun onPageStartedCallback(webView: WebView, url: String) {
+                progressBar.setVisible()
+            }
+            override fun onPageFinishedCallback(webView: WebView, url: String) {
```
Comment (kdeakinstructure) [inline apps/student/src/main/java/com/instructure/student/features/elementary/course/ElementaryCoursePagerAdapter.kt:72]: empty line should be here

**Review by kdeakinstructure — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**